api/config/config_test.go

```diff
@@ -52,9 +52,147 @@ func TestFeastServingURLs_URLs(t *testing.T) {
 	}
 }
 
-func TestStandardTransfomer_ToFeastStorageConfigs(t *testing.T) {
+func TestStandardTransformerConfig_ToFeastStorageConfigsForSimulation(t *testing.T) {
 	redisCfg := `{"is_redis_cluster": true,"serving_url":"online-storage.merlin.dev","redis_addresses":["10.1.1.10", "10.1.1.11"],"pool_size": 4,"max_retries": 1,"dial_timeout": "10s"}`
 	bigtableCfg := `{"serving_url":"10.1.1.3","project":"gcp-project","is_using_direct_storage":true,"instance":"instance","app_profile":"default","pool_size":3,"keep_alive_interval":"2m","keep_alive_timeout":"1m"}`
```

Comment:
You mentioned briefly that direct storage is not going to be supported first, is it still the case?

Reply:
from executor POV it can support both, but from service POV it can only support feast serving because we needs to fetch feature metadata